tests/integrationtest/t/expression/misc.test

```diff
@@ -181,8 +181,6 @@ set global innodb_default_row_format = dynamic;
 set global innodb_default_row_format = 'dynamic';
 SHOW VARIABLES LIKE 'innodb_default_row_format';
 SHOW VARIABLES LIKE 'character_set_server';
-SHOW VARIABLES LIKE 'innodb_file_format';
-SHOW VARIABLES LIKE 'innodb_large_prefix';
```

Comment:
Maybe keep them, to show it is no longer supported? And add the other removed ones as well?

Reply:
done in e967a1a8ce0749c4d8a060902922a1df32a18ec3